pkg/importer/avro/tests/complex_test.arrai

```diff
@@ -0,0 +1,135 @@
+//test.assert.equal(
+`##########################################
+##                                      ##
+##  AUTOGENERATED CODE -- DO NOT EDIT!  ##
+##                                      ##
+##########################################
+Avro[avro_spec="1.0", package="com.example"]:
+    # Types
```

Comment:
I don't think these section headers are necessary.

Reply:
This comes from https://github.com/anzx/acceleration/issues/328
```
sysl import required flags:

-i, --input=<SCHEMA> The schema to import
-a, --app-name=<APP-NAME> The name of the Sysl app.
sysl import optional flags:

-p, --package=<PACKAGE> The name of the Sysl package.
-o, --output=<OUT.SYSL> The output filename.
```